scripts/scripts.js

```diff
@@ -137,13 +139,40 @@ function buildBreadcrumb(main) {
   }
 }
 
+function renderMessage() {
+  const currentTime = new Date();
+  const currentHour = currentTime.getHours();
+  const splitAfternoon = 12; // 24hr time to split the afternoon
+  const splitEvening = 17; // 24hr time to split the evening
+  let message = 'Good Morning';
+  if (currentHour >= splitAfternoon && currentHour <= splitEvening) {
+    // Between 12 PM and 5PM
+    message = 'Good Afternoon';
+  }
+  if (currentHour >= splitEvening) {
+    // Between 5PM and Midnight
+    message = 'Good Evening';
+  }
+  // Between dawn and noon
+  return message;
+}
+
+function setGreetingMessage(main) {
+  const banner = main.querySelector('div');
```

Comment:
1. The condition is quite open " main > div > h1 ", instead use "first h1 in main"
2. Add null check

Reply:
@mkbansal1  done